### ISSUE TITLE
Undefined method error on Rails 7.2

### ISSUE THREAD

**Issue by arni1981** [rails/solid_cache#178]
I'm getting `undefined method 'model' for class SolidCache::Entry` on Rails 7.2.

I tried it on a brand new Rails project and just did `gem "rails", github: "rails/rails", branch: "7-2-stable"`

It happens whether I do `<%= cache%>` in a view or `Rails.cache.fetch`

**Comment by MaksJS** [comment]
A possible quick & dirty fix (to be tested):

```ruby
ActiveSupport.on_load(:solid_cache_entry) do
  class << self
    def model = self
    def scope_for_create = {}
  end
end
```

**Comment by npezza93** [comment]
This is fixed now that #179 is merged

**Comment by arni1981** [comment]
Ok, I will close it then, since it is working when pointing to the main branch of `solid_cache`. it has not been released yet though.
